python/rpdk/java/codegen.py

```diff
@@ -324,6 +341,15 @@ def write_with_relative_path(path):
         write_with_relative_path(jar)
         write_with_relative_path(project.root / "pom.xml")
 
+        # Require java plugin to be 1.1+
+        plugin_version = self._get_plugin_version(project)
+        if plugin_version < "1.1":
+            raise PluginVersionNotSupportedError(
+                "Not support aws-cloudformation-rpdk-java-plugin 1.0\n"
```

Comment:
it would be nice to fix the grammar here

Reply:
I'm not sure why this check is even occurring. The plugin doesn't need to check itself for what version it is and then throw validation errors.